src/app/pipes/longitude.pipe.ts

```diff
@@ -0,0 +1,23 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'longitude'
+})
+export class LongitudePipe implements PipeTransform {
+
+  transform(value: any, args?: any): any {
+    /*
```

Comment:
Likewise, it appears you don't use this anymore, can this commented section be cleaned?

Reply:
Same conversion than in `LatitudePipe`, I felt like if I kept the explanation in one file I would rather keep it on the other one aswell.